app/src/main/java/com/movies/presentation/home/view_model/HomeViewModel.kt

```diff
@@ -29,6 +29,9 @@ class HomeViewModel(
     val fetchFavouriteMoviesLivedata by LiveDataDelegate<List<MovieUIModel>>()
     private val categoryStateLiveData = MutableLiveData(CategoryType.POPULAR)
 
+    init {
+        getMovies()
+    }
```

Comment:
As I heard it will be better to use onCreate instead of init block, because it will be better coverable with unit tests

Reply:
done